nimble/host/services/audio/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef H_BLE_AUDIO_SVC_BASS_
+#define H_BLE_AUDIO_SVC_BASS_
+
+#include <stdint.h>
+#include "host/audio/ble_audio_bsnk.h"
+#include "syscfg/syscfg.h"
+
+#define BLE_SVC_AUDIO_BASS_UUID16                                           0x184F
+#define BLE_SVC_AUDIO_BASS_CHR_UUID16_BAS_CONTROL_POINT                     0x2BC7
+#define BLE_SVC_AUDIO_BASS_CHR_UUID16_BROADCAST_RECEIVE_STATE               0x2BC8
+
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOTE_SCAN_STOPPED             0x00
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOTE_SCAN_STARTED             0x01
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_ADD_SOURCE                      0x02
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_MODIFY_SOURCE                   0x03
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_SET_BROADCAST_CODE              0x04
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOVE_SOURCE                   0x05
+
+enum ble_svc_audio_bass_big_enc {
+    BLE_SVC_AUDIO_BASS_BIG_ENC_NOT_ENCRYPTED,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BROADCAST_CODE_REQ,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_DECRYPTING,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BAD_CODE
+};
+
+enum ble_svc_audio_bass_pa_sync {
+    BLE_SVC_AUDIO_BASS_PA_SYNC_DO_NOT_SYNC,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_AVAILABLE,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_NOT_AVAILABLE
+};
```

Comment:
This does not seem to be used

Reply:
This is for user, so they don't compare magic numbers. Now this enum is used instead of magin numbers in `modify_source` event